### ISSUE TITLE
[QUESTION] Use of Hash(#) in URL to navigate specific section

### ISSUE THREAD

**Issue by ram231** [lulupointu/vrouter#133]
Hi, I'd like to know if VRouter can navigate to specific section when a hash is added to url.

Just like how this following markdown works in a README file

# Heading 1
### Heading 3

EDIT: For more concrete example something like this: https://github.com/lulupointu/vrouter#getting-started

This is just a question that cannot be found in the official documentation but I think this can be also part of [flutter/uxr](https://github.com/flutter/uxr) discussion.

**Comment by lulupointu** [comment]
I am sorry I thought I answered this question. I apologies for the delay.

VRouter does **not** handle this natively. I believe this *could* be implemented however I don't have much time right now. I will keep this issue open and come back to it when I have more time on my hands.

In the meantime, this can be implemented manually, though I would use query parameters as they are already well integrated into `VRouter`. If you need help I would be happy to, just tell me

**Comment by lulupointu** [comment]
I am happy to report that this feature is implemented on `v1.2.0+14`!

Here is a small example:

```dart
import 'dart:math';

import 'package:flutter/material.dart';
import 'package:vrouter/vrouter.dart';

void main() {
  runApp(MyApp());
}

class MyApp extends StatelessWidget {
  const MyApp({Key? key}) : super(key: key);

  @override
  Widget build(BuildContext context) {
    return VRouter(
      // darkTheme: ThemeData(
      //   brightness: Brightness.dark,
      // ),
      // themeMode: ThemeMode.dark,
      initialUrl: '/anchors',
      routes: [
        VWidget(path: '/anchors', widget: AnchorsScreen()),
      ],
    );
  }
}

class AnchorsScreen extends StatefulWidget {
  const AnchorsScreen({Key? key}) : super(key: key);

  @override
  State<AnchorsScreen> createState() => _AnchorsScreenState();
}

class _AnchorsScreenState extends State<AnchorsScreen> {
  static Color get randomColor => Color(
        (Random().nextDouble() * 0xFFFFFF).toInt(),
      ).withOpacity(1.0);

  List<Color> colors = List.generate(20, (_) => randomColor);

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      body: SingleChildScrollView(
        child: Column(
          children: [
            for (var i = 0; i < colors.length; i++) ...[
              VAnchor(
                hoveringOverlayAlignment: Alignment.bottomRight,
                hash: 'Anchor number: $i',
                child: Padding(
                  padding: const EdgeInsets.all(8.0),
                  child: Text('Click me to go to anchor $i'),
                ),
                duration: Duration(milliseconds: 200),
              ),
              Container(
                height: 200,
                color: colors[i],
              ),
            ],
          ],
        ),
      ),
    );
  }
}
```

`VAnchor` is customizable of course, and you don't even have to use it if you simply want the hash.

Hope you like it :)

**Comment by ram231** [comment]
Thanks! I'd like to test this in a sample project. This was helpful for documentation or navigate to a specific section of a long page.
